dot/core/messages_test.go

```diff
@@ -114,9 +117,16 @@ func TestServiceHandleTransactionMessage(t *testing.T) {
 	testEmptyHeader := types.NewEmptyHeader()
 	testExtrinsic := []types.Extrinsic{{1, 2, 3}}
 
-	runtimeMock := new(mocksruntime.Instance)
-	runtimeMock2 := new(mocksruntime.Instance)
-	runtimeMock3 := new(mocksruntime.Instance)
+	runtimeMock := mocksruntime.NewInstance(t)
+	runtimeMock2 := mocksruntime.NewInstance(t)
+	runtimeMock3 := mocksruntime.NewInstance(t)
```

Comment:
Also please review #2500 when you got the time, this feels like duplicate work 😄

Reply:
Oh nice will do! Yeah working on this made me realise how much the mocks needed to be cleaned up